Resources/Prototypes/Catalog/Fills/Lockers/engineer.yml

```diff
@@ -82,6 +82,8 @@
     contents:
       - id: ClothingOuterHardsuitAtmos
       - id: ClothingMaskGasAtmos
+      - id: OxygenTankFilled
```

Comment:
Remove these. They can use tank dispensers. If dispensers aren't mapped, we will fix that.